tests/php/Extension/FluentExtensionTest.php

```diff
@@ -289,12 +289,16 @@ public function testLocalisedCopyClassNameChange(): void
      * @param string[] $expected
      * @group exclude-from-travis
      */
-    public function testLocalisedFieldsCanBeSorted($locale, array $sortArgs, $expected)
+    public function testLocalisedFieldsCanBeSorted($locale, array $sortArgs, $expected, $useOrderBy = false)
     {
-        FluentState::singleton()->withState(function (FluentState $newState) use ($locale, $sortArgs, $expected) {
+        FluentState::singleton()->withState(function (FluentState $newState) use ($locale, $sortArgs, $expected, $useOrderBy) {
             $newState->setLocale($locale);
 
-            $records = LocalisedParent::get()->sort(...$sortArgs);
+            if ($useOrderBy) {
```

Comment:
[nitpick] Use ternary operator.

Reply:
That's subjective, so unless there's some specific standard that this module adheres to which requires the ternary here I'm inclined to not make this change in this PR.

Reply:
Agree, hence the nitpick flag. Happy to keep it as is.